Api/Data/MarketplacerSellerInterface.php

```diff
@@ -188,4 +190,34 @@ public function getSourceCode(): string;
      * @return self
      */
     public function setSourceCode(string $sourceCode): self;
+
+    /**
+     * Set base domestic shipping cost applied at the Seller level
+     *
+     * @param float $value
+     * @return self
+     */
+    public function setBaseDomesticShippingCost(float $value): self;
+
+    /**
+     * Get base domestic shipping cost applied at the Seller level
+     *
+     * @return float
+     */
+    public function getBaseDomesticShippingCost(): float;
+
+    /**
+     * Set base international shipping cost applied at the Seller level
+     *
+     * @param float $value
+     * @return self
+     */
+    public function setBaseDomesticShippingFreeThreshold(float $value): self;
+
+    /**
+     * Set base international shipping cost applied at the Seller level
+     *
+     * @return float
+     */
+    public function getBaseDomesticShippingFreeThreshold(): float;
```

Comment:
It seems like the comment `international shipping cost` is incorrect. Isn't it `domestic shipping fee threshold`?

Reply:
@ducktyper fixed